content/en/metrics/nested_queries.md

```diff
@@ -0,0 +1,282 @@
+
+ {{< img src="/metrics/nested_queries/higher-res-query-example.png" alt="example of standard deviation with nested queries in the UI" style="width:100%;" >}}
+
+{{% collapse-content title="Higher resolution example query" level="h5" %}}
+
+Historically when querying a metric over the past month, you would see data at 4-hour granularity by default. You can now use nested queries to access higher granularity data over this historical timeframe. Here's an example query graphed over the past month where the query batch count is initially rolled up in 5 minute intervals. Then multilayer time aggregation is applied to calculate the standard deviation in time of this nested query over 4 hour intervals for a more human-readable graph.
+
+_Note: We recommend you define your initial rollup with the most granular rollup interval and use multilayer time aggregation with coarser rollup intervals to get more user-readable graphs._
+
+In the UI or JSON tab, it would look as follows:
+
+{{< img src="/metrics/nested_queries/nested-queries-higher-res-ui.png" alt="example of higher resolution queries using nested queries in the UI" style="width:100%;" >}}
+
+{{< img src="/metrics/nested_queries/nested-queries-higher-res-json.png" alt="example of higher resolution queries using nested queries in the JSON" style="width:100%;" >}}
+{{% /collapse-content %}} 
+
+## Moving rollup
+
+Datadog provides a `moving_rollup` function that enables aggregation of data points over a specified time window (quick refresher [here][10]). By using nested queries, you can extend its functionality to incorporate lookback mode, allowing you to analyze data points beyond the original query window. This provides a more comprehensive view of your query's trends and patterns over the specified time window.
+
+{{< img src="/metrics/nested_queries/moving-rollup-diagram.png" alt="example of old vs. new moving_rollup function" style="width:100%;" >}}
+
+The existing version of the `moving-rollup` function only supports the following aggregators:
+
+- `avg`
+- `sum`
+- `min`
+- `max`
+- `median`
+
+When nesting queries, only the lookback mode version of the `moving_rollup` function can be used. This version of the function supports the following aggregators:
+
+- `avg`
+- `sum`
+- `min`
+- `max`
+- `count`
+- `arbitrary percentile pxx` (`p78, p99, p99.99, etc.`)
+- `stddev`
+
+{{% collapse-content title="Moving rollup example query" level="h5" %}}
+When nesting these `moving_rollups`, the rollup intervals provided must get larger. 
+
+
+{{% /collapse-content %}} 
+
+
+## Remapping functions
+
+Remap functions give you the ability to refine and transform query results based on specific conditions, extending functionality for monitoring and analysis (read more about remap functions [here][]). Nested queries unlocks the following 3 new functions:
+
+- `is_greater` (`<QUERY>, <THRESHOLD>`)
```

Comment:
Can we make this a table where the three functions are each a row and then we have the parameters as well as a definition of what the function does? 

For example is_greater(<query>, 10) will return values of 1 where your metric query > 10 and 0 elsewhere